src/Business Foundation/App/NoSeries/src/Single/NoSeriesImpl.Codeunit.al

```diff
@@ -189,6 +190,7 @@ codeunit 304 "No. Series - Impl."
         if (NoSeriesLine."Line No." <> 0) and (NoSeriesLine."Series Code" = NoSeriesCode) and (NoSeriesLine."Starting Date" = NoSeriesLine2."Starting Date") then begin
             NoSeriesLine.CopyFilters(NoSeriesLine2);
             NoSeriesLine.SetRange("Line No.", NoSeriesLine."Line No.");
+            NoSeries.OnGetNoSeriesLineOnBeforeFindLast(NoSeriesLine);
```

Comment:
This is not needed as we copy all filters from NoSeriesLine2, including the filters you would have set in the previous call.

Reply:
You are right, only the first one is needed, I had included it because the RaiseObsoleteOnNoSeriesLineFilterOnBeforeFindLast was present there.